pkg/controller/deployment.go

```diff
@@ -302,7 +302,7 @@ func (dc *controller) getMachineDeploymentForMachine(machine *v1alpha1.Machine)
 		// No controller owns this Machine.
 		return nil
 	}
-	if controllerRef.Kind != "MachineDeployment" { //TODO: Remove hardcoded string
+	if controllerRef.Kind != "MachineSet" { //TODO: Remove hardcoded string
```

Comment:
I understand the change here. But just wondering if it was not leading to a bug earlier?

Reply:
I think this was a bug earlier, fixed with this.